core/src/main/java/bisq/core/dao/governance/voteresult/VoteResultException.java

```diff
@@ -28,17 +28,17 @@
 
 public class VoteResultException extends Exception {
     @Getter
-    private final Cycle cycle;
+    private final int heightOfFirstBlock;
```

Comment:
The name `heightOfFirstBlock` is a bit confusing without the context that it's the first block in the cycle

Reply:
True. Will update wit next PR....